uui-components/src/inputs/__tests__/Checkbox.test.tsx

```diff
@@ -103,4 +120,14 @@ describe('Checkbox', () => {
         expect(onBlur).toHaveBeenCalled();
         expect(input).not.toHaveFocus();
     });
+
+    it('when state equals isInvalid: true, Checkbox_container must have a \'uui-invalid\' class', async () => {
```

Comment:
Such cases better check via snapshot tests, just render Checkbox with invalid state. I guess we can add this case to the snapshot in UUI.

Reply:
done